docs/api-docs/storefront/graphql/graphql-faceted-textual-search.md

```diff
@@ -0,0 +1,982 @@
+                  value
+                }
+              }
+            }
+          }
+        }
+         filters {
+           edges {
+             node {
+               __typename
+               name
+               isCollapsedByDefault
+               ... on CategorySearchFilter {
+                 name
+                 displayProductCount
+                 isCollapsedByDefault
+                 categories {
+                   edges {
+                     node {
+                       entityId
+                       isSelected
+                       productCount
+                       subCategories {
+                         edges {
+                           node {
+                             entityId
+                             name
+                           }
+                         }
+                       }
+                     }
+                   }
+                 }
+               }
+               ... on BrandSearchFilter {
+                 name
+                 displayProductCount
+                 isCollapsedByDefault
+                 brands {
+                   edges {
+                     node {
+                       entityId
+                       name
+                       isSelected
+                       productCount
+                     }
+                   }
+                 }
+               }
+               ... on RatingSearchFilter {
+                 name
+                 isCollapsedByDefault
+                 ratings {
+                   edges {
+                     node {
+                       value
+                       isSelected
+                       productCount
+                     }
+                   }
+                 }
+               }
+               ... on ProductAttributeSearchFilter {
+                 name
+                 displayProductCount
+                 isCollapsedByDefault
+                 attributes {
+                   edges {
+                     node {
+                       value
+                       isSelected
+                       productCount
+                     }
+                   }
+                 }
+               }
+               ... on PriceSearchFilter {
+                 name
+                 isCollapsedByDefault
+                 selected {
+                   minPrice
+                   maxPrice
+                 }
+               }
+               ... on OtherSearchFilter {
+                 name
+                 displayProductCount
+                 isCollapsedByDefault
+                 freeShipping {
+                   isSelected
+                   productCount
+                 }
+                 isInStock {
+                   isSelected
+                   productCount
+                 }
+                 isFeatured {
+                   isSelected
+                   productCount
+                 }
+               }
+             }
+           }
+         }
+      }
+    }
+  }
+}
+```
+
```

Comment:
and here, `&nbsp;` if nesting not desired.

you the picture.

Reply:
Used new feature to nest while choosing the tab name